integration/SimpleCacheClient.test.ts

```diff
@@ -67,31 +67,31 @@ describe('SimpleCacheClient.ts Integration Tests', () => {
     );
     await momento.deleteCache(cacheName);
   });
-  // TODO: deprecating credentials file for now
-  // it('should use the default auth token from ~/.momento/credentials.toml', async () => {
-  //   createSystemCredentials();
-  //   const cacheName = v4();
-  //   const momento = new Momento();
-  //   await momento.createCache(cacheName);
-  //   await expect(momento.createCache(cacheName)).rejects.toThrow(
-  //     AlreadyExistsError
-  //   );
-  //   await momento.deleteCache(cacheName);
-  //
-  //   removeSystemCredentials();
-  // });
-  //
-  // it('should use the MOMENTO_PROFILE auth token from ~/.momento/credentials.toml', async () => {
-  //   createSystemCredentials('profile2');
-  //   const cacheName = v4();
-  //   const momento = new Momento();
-  //   await momento.createCache(cacheName);
-  //   await expect(momento.createCache(cacheName)).rejects.toThrow(
-  //     AlreadyExistsError
-  //   );
-  //   await momento.deleteCache(cacheName);
-  //   removeSystemCredentials();
-  // });
+
+  it('should use the default auth token from ~/.momento/credentials', async () => {
```

Comment:
Just commenting here, but it looks like integration tests are still failing in our CI environment. Is there a way we could rework how our integration tests are ran so that things still pass?

Perhaps we could set an environment variable that only exists in a CI environment, check for its existence, and if that's the case we just pull in some credentials from our environment like we do with our other repos?